.travis.yml

```diff
@@ -6,7 +6,7 @@ matrix:
     - python: "3.6-dev"
       env: TEST_CMD="flake8"
     - python: "3.5"
-      env: TEST_CMD="./tests/mypy_test.py"
+      env: TEST_CMD="./tests/mypy_test.py && ./tests/mypy_selftest.py"
```

Comment:
It would be nice to have a different job, but I don't really like the current setup where we use python versions as the labels, but really the different Python versions do completely different things (running flake, mypy, and pytype). At least one PR contributor was confused because their PR was failing in "2.7" when it was really failing in pytype. Not sure how to achieve that in Travis though; I'll just use 3.6 for now.